### ISSUE TITLE
A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_ColourContrast: Colour contrast ratio of image strips of Database size section is less than 3:1

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9265]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yWCAG2.1;#A11yTCS;#SQL Azure Data Studio;#DesktopWeb;#Win32;#MAS1.4.11;#Benchmark;#A11y_SQLAzureDataStudioBenchmark;#A11ySev2;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

Additional Details:
MAS reference: MAS1.4.11

**Repro Steps:**
1. Launch Azure Data Studio.
2. Connect to server.
3. In the Connections pane, right-click on a server & select Manage.
4. Launch Colour contrast analyser tool 
5. Check the ratio of image graph strips present under Database size section.

**Actual:**
Colour contrast ratio of Database size section’s image strips are less than required range i.e. 3:1 

**Expected:**
Colour contrast ratio of Database size section’s image strips should be greater than or equal to 3:1

**User Impact:**
Low vision users will face difficulty if image strips are less than required colour contrast ratio.

**Attachment link reference:**
[9265_A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_ColourContrast_2.zip](https://github.com/microsoft/azuredatastudio/files/4274869/9265_A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_ColourContrast_2.zip)

**Comment by alanrenmsft** [comment]
This is a charting issue, I was told when we have got for exception when this feature was tested by accessibility team before GA of ADS. I am closing this as won't fix.

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_WCAG2.1;

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.41.1
![image](https://user-images.githubusercontent.com/85863999/217253983-3ce58943-f5ed-4976-a1ec-1d71fd013972.png)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
